src/components/form/passwordPolicyComponent.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+import { Label } from './formControlsComponent';
+
+import { withTheme, withI18n } from '../widget/widgetContext';
+
+const RoundCheckbox = withTheme(styled(({ className, ...props }) =>
+    <div className={className}>
+        <input type="checkbox" checked={props.checked} onChange={() => { }} />
+        <label />
+    </div>
+)`
+    position: relative;
+    margin-right: 5px;
+
+    > label {
+        position: absolute;
+        top: 0;
+        left: 0;
+        height: 15px;
+        width: 15px;
+        background-color: #fff;
+        border: 1px solid #ccc;
+        border-radius: 50%;
+    }
+
+    > label:after {
+        position: absolute;
+        top: 4px;
+        left: 3px;
+        height: 3px;
+        width: 7px;
+        border: 1px solid #fff;
+        border-top: none;
+        border-right: none;
+        content: "";
+        opacity: 0;
+        transform: rotate(-45deg);
+    }
+
+    > input[type="checkbox"] {
+        visibility: hidden;
+    }
+
+    > input[type="checkbox"]:checked + label {
+        background-color: ${props => props.theme.get('primaryColor')};
+        border-color: ${props => props.theme.get('primaryColor')};
+    }
+
+    > input[type="checkbox"]:checked + label:after {
+        opacity: 1;
+    }
+`);
```

Comment:
The base CSS should be in a separate file to my mind.

Reply:
We're not separating CSS and JS. All the code used by a component is stored in the same file. That's why we're using https://styled-components.com/.